<ui version="4.0" >
 <class>MIPRotDialog</class>
 <widget class="QDialog" name="MIPRotDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MIP Rotation</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_Slices" >
     <property name="title" >
      <string>Recording</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Number of images per rotation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="spinBox_Images" >
           <property name="minimum" >
            <number>36</number>
           </property>
           <property name="maximum" >
            <number>7200</number>
           </property>
           <property name="value" >
            <number>360</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Degree" >
           <property name="text" >
            <string>(1° per image)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Renderer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QFrame" name="frame" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_Ortho" >
           <property name="text" >
            <string>Orthographic Projection</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_Persp" >
           <property name="text" >
            <string>Perspective Projection</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Stereo" >
        <property name="text" >
         <string>Stereo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Stereo Eye Disparity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_EyeDist" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>20</number>
           </property>
           <property name="value" >
            <number>3</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_EyeDist" >
           <property name="text" >
            <string>3 °</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_NoLOD" >
        <property name="text" >
         <string>Disable LOD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MIPRotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MIPRotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>300</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Images</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MIPRotDialog</receiver>
   <slot>UpdateDegreeLabel()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>111</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel" >
     <x>287</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Stereo</sender>
   <signal>clicked()</signal>
   <receiver>MIPRotDialog</receiver>
   <slot>UpdateStereoCheckbox()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>52</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>62</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_EyeDist</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MIPRotDialog</receiver>
   <slot>UpdateEyeDistLabel()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>194</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel" >
     <x>265</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UpdateDegreeLabel()</slot>
  <slot>UpdateStereoCheckbox()</slot>
  <slot>UpdateEyeDistLabel()</slot>
 </slots>
</ui>
